src/ctia/bundle/core.clj

```diff
@@ -453,8 +454,9 @@
                                                   filters-map
                                                   identity-map
                                                   {:limit limit
-                                                   :sort [{(name related-to) "desc"}
-                                                          {"timestamp" "desc"}]}))))
+                                                   :sort [(name related-to)
+                                                          {"timestamp" "desc"}
+                                                          "id"]}))))
```

Comment:
if you use `id` I assume that you do not need `timestamp`.

Reply:
I still need timestamp to sort entities from newest to oldest

Reply:
Why do we need to sort from newest to oldest?

Reply:
that was the original logic that got carried from here - https://github.com/threatgrid/ctia/pull/1285/files#diff-f53321c0a5e441e3481b32d1d20af1f87c9232583d527b9fe9dc468d4d9244b4L405-L413